solr/core/src/java/org/apache/solr/handler/DumpRequestHandler.java

```diff
@@ -127,4 +135,19 @@ public void init(NamedList<?> args) {
       if (nl!=null) subpaths = nl.getAll("subpath");
     }
   }
+
+  @Override
+  public Name getPermissionName(AuthorizationContext request) {
+    if (solrCore != null && solrCore.getSolrConfig().isEnableRemoteStreams()) {
+      log.warn("Dump request handler requires config-read permission when remote streams are enabled");
+      return Name.CONFIG_READ_PERM;
```

Comment:
This shuold increase security a bit, in case remote streaming is enabled.. Although, if someone has permission to switch on remote streaming, they would likely also have the config-edit and config-read permissions...